docs: adding postcss plugin load order explanation (#10803)
vitejs/vite: docs/guide/features.md

If the project contains valid PostCSS config (any format supported by [postcss-load-config](https://github.com/postcss/postcss-load-config), e.g. `postcss.config.js`), it will be automatically applied to all imported CSS.

> When you add option `postcss.plugins`, CSS minify will run after your plugin and use `build.cssTarget`. So, if you want to guarantee the consistency of your results, be aware of this.

Reviewer: ```suggestion
Note that CSS minification will run after PostCSS and will use [`build.cssTarget`](/config/build-options.md#build-csstarget) option.
```
